### DISCUSSION TITLE
Is there a way to just display my total commits?

### DISCUSSION THREAD

**Discussion by WedleySilva** [anuraghazra/github-readme-stats#3511] [Q&A] [answered]
I would like to know if there is any way for me to display only the total number of commits when importing the stats.
Thanks.

**Comment by qwerty541** [comment, marked as answer]
Hey, @WedleySilva! You can achieve the desired result with a combination of `&hide_rank=true` and `&hide=contribs,prs,issues,stars` query parameters. I added an example below.

```html
<img src="https://github-readme-stats.vercel.app/api?username=WedleySilva&include_all_commits=true" />
```

<img src="https://github-readme-stats.vercel.app/api?username=WedleySilva&include_all_commits=true" />

```html
<img src="https://github-readme-stats.vercel.app/api?username=WedleySilva&hide=contribs,prs,issues,stars&hide_rank=true&include_all_commits=true" />
```

<img src="https://github-readme-stats.vercel.app/api?username=WedleySilva&hide=contribs,prs,issues,stars&hide_rank=true&include_all_commits=true" />

**Reply by WedleySilva** [reply]
@qwerty541 Thank you very much for your help.
